core/infrastructure/keycloak/v1alpha1/register.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package v1alpha1 contains API Schema definitions for the keycloak v1alpha1 API group
+// +kubebuilder:skip
+// +k8s:deepcopy-gen=package,register
+// +groupName=keycloak.org
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "keycloak.org", Version: "v1alpha1"}
+
+	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
+	SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
+	// AddToScheme ...
+	AddToScheme = SchemeBuilder.AddToScheme
```

Comment:
Same here. We should try to import from their modules.

Reply:
To upgrade operator-sdk we need to bump controller-runtime to v0.8.3 and kubernetes dependencies to v0.20.2.
Many of 3rd party operator dependency doesn't allow us to do so because they need lower/higher version of these dependencies